src/core/utils/helpers/url.ts

```diff
@@ -102,3 +102,12 @@ export const turnUrlStringsIntoObjects = (urls: { [key: string]: string }) => {
     {}
   );
 };
+
+export const replaceCurrentLocation = (replacementUrl: URL) => {
+  window.history.replaceState(null, "", replacementUrl);
+};
+
+export const removeQueryParametersFromUrl = (url: URL, parameter: string) => {
+  url.searchParams.delete(parameter);
+  return url;
+};
```

Comment:
Consider if this function makes sense.

It contains a single line of code without much complexity, additional values etc. To me it seems as if developers might as well call `searchParams.delete` directly.

Reply:
Yeah I had the same thought. But since we have abstracted all URL handling away in other cases I ended up thinking we should do that here as well.